src/cuttlefish.erl

```diff
@@ -52,6 +52,10 @@ otp(DesiredMinimumOTPVersion, IfGreaterOrEqual, IfLessThan) ->
 otp([], []) ->
     %% They're the same length AND all previous chars were matches
     true;
+otp([$R|TMin], TVer) ->
+    otp(TMin, TVer);
+otp(TMin, [$R|TVer]) ->
+    otp(TMin, TVer);
```

Comment:
Minor nit-pick: The 'T' in 'TMin' and 'TVer' stands for the tail. So these could be renamed to 'Min' and 'Ver', if they are just the actual values.